consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/CliqueBlockHashing.java

```diff
@@ -23,6 +23,7 @@
 import java.util.function.Supplier;
 
 public class CliqueBlockHashing {
+  private static final Address ADDRESS_ZERO = Address.extract(Hash.ZERO);
```

Comment:
I suspect we should just introduce `Address.ZERO = Address.fromHexString('0x0')` in the `Address` class.

Reply:
done